cpp/src/arrow/flight/sql/odbc/flight_sql/flight_sql_driver.cc

```diff
@@ -63,45 +67,15 @@ odbcabstraction::Diagnostics& FlightSqlDriver::GetDiagnostics() { return diagnos
 void FlightSqlDriver::SetVersion(std::string version) { version_ = std::move(version); }
 
 void FlightSqlDriver::RegisterLog() {
-  odbcabstraction::PropertyMap propertyMap;
-  driver::odbcabstraction::ReadConfigFile(propertyMap, CONFIG_FILE_NAME);
-
-  auto log_enable_iterator = propertyMap.find(SPDLogger::LOG_ENABLED);
-  auto log_enabled = log_enable_iterator != propertyMap.end()
-                         ? odbcabstraction::AsBool(log_enable_iterator->second)
-                         : false;
-  if (!log_enabled) {
-    return;
-  }
-
-  auto log_path_iterator = propertyMap.find(SPDLogger::LOG_PATH);
-  auto log_path = log_path_iterator != propertyMap.end() ? log_path_iterator->second : "";
-  if (log_path.empty()) {
+  std::string log_level_str = arrow::internal::GetEnvVar(ODBC_LOG_LEVEL).ValueOr("");
+  if (log_level_str.empty()) {
     return;
   }
+  auto log_level = ToLogLevel(std::stoi(log_level_str));
 
-  auto log_level_iterator = propertyMap.find(SPDLogger::LOG_LEVEL);
-  auto log_level = ToLogLevel(log_level_iterator != propertyMap.end()
-                                  ? std::stoi(log_level_iterator->second)
-                                  : 1);
-  if (log_level == odbcabstraction::LogLevel_OFF) {
-    return;
-  }
-
-  auto maximum_file_size_iterator = propertyMap.find(SPDLogger::MAXIMUM_FILE_SIZE);
-  auto maximum_file_size = maximum_file_size_iterator != propertyMap.end()
-                               ? std::stoi(maximum_file_size_iterator->second)
-                               : DEFAULT_MAXIMUM_FILE_SIZE;
-
-  auto maximum_file_quantity_iterator = propertyMap.find(SPDLogger::FILE_QUANTITY);
-  auto maximum_file_quantity = maximum_file_quantity_iterator != propertyMap.end()
-                                   ? std::stoi(maximum_file_quantity_iterator->second)
-                                   : 1;
-
-  std::unique_ptr<odbcabstraction::SPDLogger> logger(new odbcabstraction::SPDLogger());
-
-  logger->init(maximum_file_quantity, maximum_file_size, log_path, log_level);
-  odbcabstraction::Logger::SetInstance(std::move(logger));
+  // Enable driver logging. Log files are not supported on Windows yet, since GLOG is not
+  // tested fully on Windows.
+  arrow::util::ArrowLog::StartArrowLog("arrow-flight-sql-odbc", log_level);
```

Comment:
Logging documentation is available at https://github.com/apache/arrow/pull/46099 (see [here](https://github.com/apache/arrow/pull/46099/commits/83c7a744606e4cf7e725c719404d3fc571710967#diff-dfecfcd0d12d6c9cca37c1b7e501584413de1f0a1e442a49faaea7caff5395d2)). We plan to add the docs in future PRs